daemon/libnetwork/store.go

```diff
@@ -127,11 +127,11 @@ retry:
 	return nil
 }
 
-func (c *Controller) networkCleanup() {
+func (c *Controller) networkCleanup(ctx context.Context) {
 	for _, n := range c.getNetworksFromStore(context.TODO()) {
```

Comment:
This still has a `TODO` context; as we're deleting things in a loop, should the loop handle context cancellation? Or should this also be non-cancellable and always continue until all networks were tried?